assets/js/bootstrap-migration/common-components/searchBar.js

```diff
@@ -0,0 +1,47 @@
+var storefrontApp = angular.module('storefrontApp');
+storefrontApp.component('vcSearchBarMigration', {
+    templateUrl: "themes/assets/js/bootstrap-migration/common-components/searchBar.tpl.html",
+    bindings: {
+        formClass: '<',
+        placeholder: '<',
+        searching: '<',
+        noResults: '<',
+        query: '@',
+        categoriesLabel: '<',
+        productsLabel: '<',
+        submitLabel: '<',
+        categoryLimit: '@',
+        productLimit: '@'
+    },
+    controller: ['$scope', '$q', 'catalogService', function ($scope, $q, catalogService) {
```

Comment:
Try to reuse / inherit this controller. If can't do it quickly, resolve with "won't fix"